docs/cosmos-hub/what-is-gaia.md

```diff
@@ -0,0 +1,22 @@
+# What is Gaia?
```

Comment:
this section should also go into the repo's main `README`, people get confused between Gaia and the Cosmos Hub

Reply:
Absolutely, we need a clear explainer in the README